src/modules/labels/Label.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Label.css';
+
+const propTypes = {
+    children: PropTypes.oneOfType([
+        PropTypes.node,
+        PropTypes.string,
+    ]).isRequired,
+};
+
+const Label = ({ children }) => (
+    <div className="j-label">
+        { children }
+    </div>
+);
+
+Label.propTypes = propTypes;
```

Comment:
I started to notice this but maybe it would be simpler to just declare the prototypes here instead of assigning them after declaring.